### DISCUSSION TITLE
Multi GPUS Setup

### DISCUSSION THREAD

**Discussion by imedjendoubi** [lllyasviel/Fooocus#3603] [Q&A] [answered]
How can use multigpus setup to generate images faster?

Thank you

**Comment by Rogala** [comment, marked as answer]
You can run multiple instances of the programme to work with different GPUs, but it is not possible to use the capacities for one task at the same time.

**Comment by mafju80** [comment]
but I thought multi GPU was possible under Ubuntu and Docker?
